code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -337,27 +337,53 @@ var/regex/lizard_hiSS = new("S+", "g")
 	name = "Golem"
 	id = "golem"
 	specflags = list(NOBREATH,HEATRES,COLDRES,NOGUNS,NOBLOOD,RADIMMUNE,VIRUSIMMUNE,PIERCEIMMUNE)
-	speedmod = 3
+	speedmod = 2
 	armor = 55
 	siemens_coeff = 0
 	punchdamagelow = 5
 	punchdamagehigh = 14
 	punchstunthreshold = 11 //about 40% chance to stun
-	blacklisted = 1
```

Comment:
if this isn't going to be an inherent part of golems, you should make sure to readd it to adamantine ones because they were inheriting it from here.